collectivites/templates/index.html

```diff
@@ -4,9 +4,83 @@
 
 {% block content %}
 <div class="row mwide twide">
-    <div class="col wide">
-
-        <p>Bienvenue sur le portail collectivités blah…</p>
-    </div>
+    <main class="col wide">
+        <ban></ban>
```

Comment:
`ban` est de toute façon un mauvais nommage ici: il y aura plusieurs vues ban, donc a minima je dirais `ban-xxx` (same pour le nom des fichiers).
Je préfère un bon nommage qu'un mauvais commentaire. Mais voyons à quoi on aboutit dans ces premières itérations ;)